advisor/src/main/kotlin/advisors/GitHubDefectsConfiguration.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
Should this simply be 2020 like in the source file, not going as deep as considering when the class was introduced? Same for the other extracted classes.

Reply:
Since I already took the time to figure out the correct years I'd rather keep them.

Reply:
Ok, it wasn't clear to me that you already invested that effort as the years looked wrong at the first glance. For example, I thought `OssIndexConfiguration` was introduced earlier, but you're right, it seems to have been only in 2023.